### ISSUE TITLE
[RFE] f5 testing service

### ISSUE THREAD

**Issue by DumpySquare** [f5devcentral/f5-conx-core#17]
Looking at creating a testing service to minic all api endpoints.  Having to run an a bigip to complete most of the tests is cumbersome.  

A small, stand-alone test rest server/service should accelerate feature deployment, and full fill testing needs with other projects like vscode-f5 extension and the cli where end-to-end tests are severely lacking

initial:
- bigip token endpoint
- tmos info/license endpoint details
- as3 endpoints
  - basically mimic the "dry-run" parameter
  - just receive declaration (possibly validate) and return appropriate response
  - hold deployed declarations to support GET and DELETE operations

future:
- do/cf/ts/fast endpoints
- tmos endpoints
  - file upload/download
  - qkview/ucs/atc-service

Using this method could easily allow me to mock up new services like Volterra/NIM2.0/DWAF with just postman collections (with responses)

**Comment by DumpySquare** [comment]
cbip token working, 90% on needed AS3 functionality for these projects
https://github.com/DumpySquare/f5-mock-service
